### PR TITLE
simiplify this build using our autoconf pipeline

### PR DESCRIPTION
**Pull request by dustinkirkland**
Use our autoconf pipeline as much as possible

### CODE REVIEW

**Comment by dustinkirkland** [comment]
@EyeCantCU You might want to have a look at this PR, based on your feedback on the libfastjson one...

File: liblogging.yaml [hunk 1]
```diff
@@ -26,16 +24,16 @@ pipeline:
...
+  - uses: autoconf/configure
+
   - runs: |
-      libtoolize --copy
-      aclocal -I m4 --install
-      autoupdate
-      autoheader
-      autoconf
-      automake --foreign --add-missing --force-missing --copy
       ./configure --prefix=/usr --disable-journal --enable-stdlog
```
Comment (EyeCantCU) [inline liblogging.yaml:30]: We can remove this as it's handled by the configure pipeline above it

**Review by EyeCantCU — CHANGES_REQUESTED** [review]

File: liblogging.yaml (hunk 1 above)
Comment (dustinkirkland) [reply liblogging.yaml:30]: Hmm, how to I add the specific --disable and --enable options?

Comment (dustinkirkland) [reply liblogging.yaml]: I figured it out!

**Review by EyeCantCU — APPROVED** [review]

**Pull request merged**